src/Components/Shared/Button/index.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import Styles from './button.module.css';
+
+function Button({ onCloseButton, text, img, classNameButton }) {
+  return (
+    <div onClick={onCloseButton ? onCloseButton : undefined}>
+      {img ? (
+        <img className={Styles.icon} src={img} />
+      ) : (
+        <button
+          className={
+            classNameButton === 'deleteButton'
```

Comment:
We should pass the class name as a prop and directly use that name in the className, to avoid doing this logic here.

for example: Button({(others props)... classNameButton})

and here, directly: className={classNameButton}